elements/bulbs-slideshow/bulbs-slideshow.js

```diff
@@ -103,9 +109,18 @@ class BulbsSlideshow extends BulbsHTMLElement {
 
   afterSlideTransition () {
     this.render();
+    this.sendPageView.bind(this).call();
     this.navigateToNextSlide();
   }
 
+  sendPageView () {
+    this.analyticsManager.trackPageView(
```

Comment:
Comment about duplication aside, I don't see why you need to `.bind(this)` for calls to `sendPageView`